build/quicklink.php

```diff
@@ -83,13 +83,15 @@ function quicklink_enqueue_scripts() {
 	 * @param array {
 	 *     Configuration options for Quicklink.
 	 *
-	 *     @param string[] $urls      Array of URLs to prefetch (override).
-	 *     @param string   $el        CSS selector for the DOM element to prefetch in-viewport links for.
+	 *     @param string   $el        CSS selector for the DOM element to observe for in-viewport links to prefetch.
+	 *     @param int      $limit     The total requests that can be prefetched while observing the $el container.
+	 *     @param int      $throttle  The concurrency limit for simultaneous requests while observing the $el container.
+	 *     @param int      $timeout   Timeout after which prefetching will occur.
+	 *     @param string   $timeoutFn Custom timeout function.
 	 *     @param bool     $priority  Attempt higher priority fetch (low or high). Default false.
 	 *     @param string[] $origins   Allowed origins to prefetch (empty allows all). Defaults to host for current home URL.
 	 *     @param string[] $ignores   Regular expression patterns to determine whether a URL is ignored. Runs after origin checks.
-	 *     @param int      $timeout   Timeout after which prefetching will occur.
-	 *     @param string   $timeoutFn Custom timeout function.
+	 *     @param string[] $urls      Array of URLs to prefetch.
```

Comment:
As `timeoutFn` is supported, so too should `onError` be supported, yeah?

```suggestion
	 *     @param string   $onError   Custom error handler. Must refer to a named global function in JS.
	 *     @param string[] $urls      Array of URLs to prefetch.
```